tests/datasource/fluent/data_asset/test_file_path_asset.py

```diff
@@ -87,3 +102,319 @@ def test_get_batch_list_from_batch_request__sort_descending(
     for i, batch in enumerate(batches):
         assert batch.metadata["year"] == str(expected_years[i])
         assert batch.metadata["month"] == str(expected_months[i])
+
+
+@pytest.fixture
+def datasource(mocker):
+    return mocker.Mock(spec=PandasFilesystemDatasource)
+
+
+@pytest.fixture
+def file_path_data_connector(mocker):
+    return mocker.Mock(spec=FilePathDataConnector)
+
+
+@pytest.fixture
+def asset(request, datasource, file_path_data_connector) -> _FilePathDataAsset:
+    # asset = _FilePathDataAsset(name="test_asset", type="_sql_asset")
```

Comment:
Is this the only asset used now? It is strange that it's type is `_sql_asset` but it's a file path asset.

Reply:
This asset will be replaced by a list of all the  child assets the API is implemented on - just have this here as a placeholder for now.